package.json

```
@@ -14,12 +14,12 @@
     "polaris.shopify.com"
   ],
   "scripts": {
-    "build": "yarn workspaces run build",
+    "build": "turbo run build",
+    "dev": "turbo run dev --no-cache --parallel --continue",
```

```suggestion
    "dev": "turbo run dev --parallel",
```

You already have `cache: false` set it the `turbo.json` config for the `dev` pipeline.

https://turborepo.org/docs/reference/command-line-reference#--continue
> By default, specifying the `--parallel` flag will automatically set `--continue` to `true`